src/accelerate/data_loader.py

```diff
@@ -19,6 +19,8 @@
 import torch
 from torch.utils.data import BatchSampler, DataLoader, IterableDataset, RandomSampler
 
+from accelerate.utils import is_torchdata_stateful_dataloader_available
```

Comment:
This can now be added under the `from .utils import (` block in line 26 :)

Reply:
done